docs/design/autoware-architecture/control/index.md

```diff
@@ -1,5 +1,70 @@
+
+These are critical roles of Control stack:
+
+- Derive the lateral command, i.e. how to manipulate the steering wheel
+- Derive the longitudinal command, i.e. how to step on the pedal (gas/brake)
+- Integrate all the control signals to the vehicle interface, including:
+  - lateral command
+  - longitudinal command
+  - Turn indicators
+  - Hazard lights
+  - Gear
+  - etc..
+
+## Input
+
+The table below summarizes the overall input into Control stack:
+
+| Input              | Topic Name(Data Type)                                                                                                                                                      | Explanation                                                                                                                                                                                                                                                                                                         |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Trajectory         | `/planning/scenario_planning/trajectory`<br>(_autoware_auto_planning_msgs/msg/Trajectory_)                                                                                 | This is the local sequence of pose, twist and acceleration that Control stack must follow. This must be smooth, and kinematically possible to follow by the Control stack. By default, the trajectory is 10 seconds long at 0.1 second resolution.                                                                  |
+| Vehicle Kinematics | `/localization/kinematic_state`<br>(_nav_msgs/msg/Odometry_)                                                                                                               | This includes vehicle's pose and velocity information. Both lateral and longitudinal commands are calculated based on current velocity. For more details, refer to the following description of node implementation.                                                                                                |
+| Map data           | `/map/vector_map`<br>(_autoware_auto_mapping_msgs/msg/HADMapBin_)                                                                                                          | This includes all static information about the environment, such as: <ul><li>Lane connection information used for route planning from starting position to goal position</li><li>Lane geometry to generate reference path used to calculate trajectory </li><li> All information related to traffic rules</li></ul> |
```

Comment:
Currently, the control module(lane depature) uses HDmap information.
But I feel strange with it.
@mitsudome-r Is this a description of the actual implementation? or autoware control's role and interface?